ui/src/components/role/AddRole.js

```diff
@@ -185,9 +185,14 @@ export default class AddRole extends React.Component {
 
     addMember() {
         let name = this.state.newMemberName;
-        let date =
-            this.state.newMemberDate && this.state.newMemberDate.length > 0
-                ? this.state.newMemberDate
+        let expirationDate =
```

Comment:
Lets initialize the new state key at line 140 to keep it consistent.
newMemberDate: '', => memberExpiry: ''
and also lets add a default blank value for memberReviewReminder: ''